components/ActionButton.tsx

```diff
@@ -28,7 +31,11 @@ const ActionButton: FC<ActionButtonProps> = ({
       break
   }
 
-  return (
+  return href ? (
+    <Link href={href}>
+      <a className={classStyle}>{text}</a>
+    </Link>
+  ) : (
```

Comment:
This seems like such a different thing, that I think it would be best to build as a separate component all together; or just use the React/Link.

Reply:
This is the way it was done in the design system in order to create a "link button". I think there is merit in this being here since we can re-use the styling but we can have a discussion about it

Reply:
hmmm right, I might take a crack at separating the use while keeping the commonality; after you merge.